.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+default_language_version:
```

Comment:
Is there a way to get rid of these commits? These have been introduced by merging an earlier commit and are not part of this PR, right?